easypost/__init__.py

```diff
@@ -154,90 +153,45 @@ def convert_to_easypost_object(response, api_key, parent=None, name=None):
 
 
 def _utf8(value):
-    if six.PY2:
-        # Python2's urlencode wants bytestrings, not unicode
```

Comment:
# Reviewer Note

Talking with Avi, it appears that the Python2 encoding stuff may not be necessary here as tests pass without cassettes with all these changes.

Reply:
Or if it is necessary then we're not testing cases that matter!